spec/models/user_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
 describe User do
-  it "has a valid factory" do
-    user = FactoryGirl.create(:user)
+  it 'has a valid factory' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.